src/js/inverted-index.js

```diff
@@ -0,0 +1,138 @@
+/* eslint class-methods-use-this: "off" */
+/* eslint no-unused-vars: "off" */
+
+/** Class representing a index. */
+class Index {
+  /**
+   * Create and Index and search the index.
+   */
+  constructor() {
+    this.files = {
+      allBooks: []
+    };
+    this.words = [];
+    this.wordIndex = {};
+    this.bookNos = [];
+    this.bookTitle = [];
+    this.allBooksNos = [];
+    this.allBooksTitle = [];
+  }
+/**
+* takes a file name and removes the extension
```

Comment:
- This description is not accurate; your method below takes the file content not file name.

- You may also want to rephrase this sentence(replace the word extension with a more appropriate word)

- Nitpick: Start the sentence with a capital letter.